### ISSUE TITLE
Suggestion: Scroll-Position per item

### ISSUE THREAD

**Issue by Dragonseel** [ExileNG/NGSpaceCompany#6]
I would like to have a option to enable a saved scroll position per item.
When switching around multiple items and back to optimize/reduce some resource collection time, it can get a bit tedious to always have to scroll to the same places again.
I do not know if this is possible/feasible in any way.

What I would like is following: If I scroll to the lowest machine in for example Titanium, and see that the storage for Lunarite is too small, when I change to Lunarite scroll up to the storage-expand option and then change back to Titanium, the scroll is still on the lowest machine I was initially working on. When then returning to Lunarite because a single update was not enough my view is still on the storage-expand option where I last left Lunarite.
Ideally this behavior can quickly be toggled on/off because there are legitimate use-cases for the current behavior (building the same-tier machine in different resource types one after another).

**Comment by briangithex** [comment]
UX-Wise, I think it'd be cleaner to just have all of the component entries be collapsible into a shortened summary view (and for that state to be remembered when navigated to/from the panel).

However, a more targeted approach would be for resources to support quicknav.  Clicking a quantity in the costs section open that resource's panel and would scroll to the quantity section of the panel.  Clicking a checkmark/xmark would scroll to the capacity section of the resource panel.

Mind you, personally I'm happy as-is.  I just think your current proposal, while technically possible, is not especially user-friendly.

**Comment by Dragonseel** [comment]
Yeah, I am not a UX-Designer so it might be a bad suggestion. I just noticed that I repeatedly scroll up and down on the same few resources while trying to build my first dyson ring. I want to see which resource in the next segment is the slowest one, so I scroll up on the 'Dyson' resource, only to change to a specific resource and scroll down to the highest-level producer I have. And then repeat.
The first idea I had that would fix this, was to be able to save/restore the scroll position per resource.
A "shortcut" based navigation where clicking on the timer in a item I am waiting for and then getting to the highest producer for that resource would be cool, though on getting back to the thing I was originally building (which would need an additional feature for quickly going back) the scroll would again be "wrong". 

Just to elaborate on my thought-process. I am okay if the devs decide that this is not worth the time or does not fit the targeted UX philosophy.

**Comment by briangithex** [comment]
> A "shortcut" based navigation where clicking on the timer in a item I am waiting for and then getting to the highest producer for that resource would be cool, though on getting back to the thing I was originally building (which would need an additional feature for quickly going back) the scroll would again be "wrong".

Yeah, the keyboard navigation approach I mentioned would need to remember scroll position (whereas clicking would not care about previous scroll position).

<details>
  <summary>Spoiler</summary>
At least in the case of Dyson, this isn't *that* important.  The more times you rebirth, the narrower the window becomes where this sort of micromanagement is worth the effort.
</details>
